libfreerdp/crypto/test/TestKnownHosts.c

```diff
@@ -46,8 +46,9 @@ static int prepare(const char* currentFileV2, const char* legacyFileV2, const ch
 
 	for (i=0; i<sizeof(hosts)/sizeof(hosts[0]); i++)
 	{
-		fwrite(hosts[i], strlen(hosts[i]), sizeof(char), fl);
-		fwrite(hosts[i], strlen(hosts[i]), sizeof(char), fc);
+		if (fwrite(hosts[i], strlen(hosts[i]), sizeof(char), fl) != sizeof(char) ||
+			fwrite(hosts[i], strlen(hosts[i]), sizeof(char), fc) != sizeof(char))
+			goto finish;
```

Comment:
@bmiklautz sure, and in this case a simple if (fwrite(str, strlen(str), 1, fp) != 1) is better. And if character size > 1 I'd also want to write one element in this case using fwrite(wstr, wcslen(str)*sizeof(wchar_t), 1, fp != 1)